File: Tarea1_Arreglos_27-08-25/LEM_Arreglos.js
const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

let nums = [10, 20, 30, 40, 50];

console.log("Primer Elemento:", nums[0]);

nums[3] = 55;

console.log("arreglo inicial:");
for (let n of nums) {
  console.log(n);
}

rl.question("¿Cuantos números?: ", (answer) => {
  let Size = parseInt(answer);
  let arr = [];
  let i = 0;

  function pedirNum() {
    if (i < Size) {
      rl.question(`Ingresar Valor: ${i + 1}: `, (valor) => {
        arr.push(parseInt(valor));
        i++;
        pedirNum();
      });
    } else {
      console.log("valores ingresados:", arr);
      rl.close();
    }
  }

  pedirNum();
});

File: Tarea1_Arreglos_27-08-25/LEM_Arreglos.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() {
    int nums[5] = {10, 20, 30, 40, 50};

    cout << "Primer elemento: " << nums[0] << endl;

    nums[3] = 55;

    cout << "Array: ";
    for(int i = 0; i < 5; i++) {
        cout << nums[i] << " ";
    }
    cout << endl;

    int n;
    cout << "¿Cuantos numeros?: ";
    cin >> n;

    vector<int> arr(n); 
    for(int i = 0; i < n; i++) {
        cout << "Ingresar valor: " << (i+1) << ": ";
        cin >> arr[i];
    }

    cout << "Valores ingresados: ";
    for(int i = 0; i < n; i++) {
        cout << arr[i] << " ";
    }
    cout << endl;

    return 0;
}